packages/tools/piw-utils/src/utils/PageEditorUtils.ts

```diff
@@ -71,6 +71,17 @@ export function changePropertyIn<T, TKey extends keyof T>(
     modifyProperty(modify, propertyGroups, key, nestedPropIndex, nestedPropKey);
 }
 
+export function transformGroupsIntoTabs(properties: Properties) {
+    const groups: PropertyGroup[] = [];
+    properties.forEach(property => {
```

Comment:
`properties` isn't an array, so you don't need to iterate on it:
```
export interface Properties {
    property?: Property[];
    propertyGroup?: PropertyGroup[];
    systemProperty?: SystemProperty[];
}
```

Reply:
Are you sure?
`export type Properties = PropertyGroup[];`

Reply:
https://github.com/mendix/widgets-resources/blob/5e05eaf0c58911a2bedd6fdf950b090c3970672d/packages/tools/piw-utils/src/typings/PageEditor.ts#L1